hs_sitemap/templates/sitemap.html

```diff
@@ -0,0 +1,28 @@
+{% extends "base.html" %}
+
+{% block all_content %}
+    <style>
+    h1,h2,h3,h4 {
+        margin: 20px;
+    }
+    h3 {
+        padding-left: 20px;
+    }
+    h4 {
+        padding-left: 40px;
+    }
+    </style>
```

Comment:
@ironfroggy Is there a reason to put these style definitions in html template rather than a css file?